modules/library/render/src/test/java/org/geotools/renderer/lite/StreamingRendererTest.java

```diff
@@ -237,19 +237,9 @@ public void testDrawIntepolation() throws Exception {
         // test right interpolation hint is set on Graphics2D
         assertEquals(
                 graphics.getRenderingHint(JAI.KEY_INTERPOLATION),
-                Interpolation.getInstance(Interpolation.INTERP_BICUBIC));
```

Comment:
And here?

Reply:
Just using a "not commonly used" interpolation to make sure hints is actually using it once set (so it's not using a default)